app/templates/content/index-crown-hosting.html

```diff
@@ -39,7 +39,7 @@ <h1 class="govuk-heading-xl">
           to talk about your datacentre hosting needs.
 
         </p>
-        <h2 class='summary-item-heading'>
+        <h2 class='govuk-heading-l'>
```

Comment:
Any reason why we've gone for `govuk-heading-l` and not `-m`? I think we've used medium for pretty much all the other `h2`s so far, and it more closely matches the size on production at the moment?

Reply:
Good catch! I will change it.